specification/datasets/cost_and_usage/columns/allocatedtags.md

```diff
@@ -0,0 +1,62 @@
+# Allocated Tags
+
+The Allocated Tags column represents the set of [*tags*](#glossary:tag) assigned to [*tag sources*](#glossary:tag-source) which are specifically applicable to allocated charges resulting from a provider-calculated split cost allocation.
```

Comment:
Same comment as earlier: I would really like to avoid the jargon here if we can.
```suggestion
The Allocated Tags column represents the set of [*tags*](#glossary:tag) assigned to [*tag sources*](#glossary:tag-source) which are specifically applicable to charges that have been allocated by the provider.
```